static/app/views/dashboardsV2/widgetCard/transformSessionsResponseToSeries.tsx

```diff
@@ -9,17 +9,20 @@ export function transformSessionsResponseToSeries(
   if (response === null) {
     return [];
   }
+  function getSeriesName(field: string, group: SessionApiResponse['groups'][number]) {
```

Comment:
nit: should this function be defined outside of the `transformSessionsResponseToSeries` function (but same file)?